### ISSUE TITLE
Missing second view (allow/deny)

### ISSUE THREAD

**Issue by erperejildo** [EddyVerbruggen/cordova-plugin-googleplus#471]
I'm missing second and last view on the login process. Working locally but not on pro:
https://play.google.com/store/apps/details?id=drodriguez.apps.Words

You'll find also Facebook login which works ok.
```
const loading = this.loadingCtrl.create({
      content: this.translatedMsgs['ALERT_CONNECTING']
    });
    loading.present();
    const that = this;

return this.gplus.login({
          'webClientId': '5200545345-pifsi37fsbmkn3uhaetimvlabel5l925.apps.googleusercontent.com',
          'offline': true,
          'scopes': 'profile email'
        })
          .then((res) => {
            return that.AngularFireAuth.auth
              .signInWithCredential(firebase.auth.GoogleAuthProvider.credential(res.idToken))
              .then((success) => {
                loading.dismiss();
                that.updateUserData(success);
              })
              .catch((error) => {
                loading.dismiss();
                that.showToast(error.message);
              });
          })
          .catch((error) => {
            loading.dismiss();
          });
```

And this is my Ionic info:
```
cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.19.1
    ionic (Ionic CLI) : 3.19.1

global packages:

    cordova (Cordova CLI) : 8.0.0

local packages:

    @ionic/app-scripts : 3.1.8
    Cordova Platforms  : android 7.0.0
    Ionic Framework    : ionic-angular 3.9.2

System:

    Android SDK Tools : 26.1.1
    Node              : v8.10.0
    npm               : 5.6.0
    OS                : OS X El Capitan
    Xcode             : Xcode 8.2 Build version 8C38

Environment Variables:

    ANDROID_HOME : /Users/danielrodriguez/Library/Android/sdk/

Misc:

    backend : pro
```

**Comment by codinronan** [comment]
Hi @erperejildo were you able to get that working? I noticed you updated your app today so hopefully it works for you now, and you can close the issue

**Comment by erperejildo** [comment]
Not really... I'm making some updates like removing the async function (the Facebook one wasn't like this) and also returning the error to see what's going on. But no luck at the moment...

**Comment by codinronan** [comment]
@erperejildo Hmm ok I installed your app and tried to log in w/Google. I do get the screen to pick an account, but after selecting it doesn't confirm. This tells me that you have GooglePlus properly configured, and Firebase is not. Very likely either your SHA1 is missing, or the bundle ID is missing/incorrect. Things to check.

**Comment by codinronan** [comment]
@erperejildo this is my code that has worked for quite a long time. By the way, your scopes are declared wrong - I bet that is your problem. BTW I am using an async function but promise-based should work the same way.

```
      const scopes = 'https://www.googleapis.com/auth/plus.login';
      const clientId = environment.GOOGLE_CLIENT_ID;
      const googleOptions = { webClientId: clientId, 'offline': true, scopes };

      const response = await this.googlePlus.login(googleOptions);
      const credential = firebase.auth.GoogleAuthProvider.credential(response.idToken);
      this.firebaseUser = await firebase.auth().signInWithCredential(credential);

      // success!
```

**Comment by erperejildo** [comment]
It's ok configured. That's what I mean: works locally installed on my phone (which is what I don't understand). It's quite similar to the Facebook one and that one works ok also on production.

Your configuration is the same, the only difference is the scope (not sure about that) and the async function as you said. I also had that before but it wasn't working either so I removed it to try a different way.

If my scopes are wrong, why is working locally? what's the difference? 
I followed this tutorial: https://angularfirebase.com/lessons/ionic-google-login-with-firebase-and-angularfire/

**Comment by codinronan** [comment]
If you dig into the angular code it ignores scopes it doesn't understand. Since profile and email are included by default, that's why you get what you want locally.

Working locally, but not remotely, suggests that you have your dev SHA included on Firebase, but not the production signing certificate SHA.

**Comment by erperejildo** [comment]
Didn't work either...

How can I get the production signing certificate SHA?

**Comment by codinronan** [comment]
Hey @erperejildo, so basically when you package your app, it will be signed with a signing certificate. On dev, android does this for you; for it to work, you had to use openssl to extract the SHA from the certificate. 

When you package your app for production, you have to do the same, but the Android tools don't provide a certificate for you; you have to generate one. If you selected your app for "Verified by Google" when you submitted it, then Google has signed it for you using its own signing certificate. It is that SHA that must be added to the Firebase dashboard under Settings->Apps->Android. 

Let me know how you have signed your app, and I can tell you where to look

**Comment by erperejildo** [comment]
I prepare my apk with this command (this includes the whole thing):
`ionic cordova build --pro --release android && jarsigner -verbose -sigalg SHA1withRSA -digestalg SHA1 -keystore resources/firma_words.keystore platforms/android/app/build/outputs/apk/release/app-release-unsigned.apk words && /Users/danielrodriguez/Library/Android/sdk/build-tools/26.0.2/zipalign -v 4 platforms/android/app/build/outputs/apk/release/app-release-unsigned.apk Words.apk`

Also on Firebase I have a SHA certifica where you mention (`A1:C4:24:13:BE:68:80:`..)

**Comment by erperejildo** [comment]
Is this related?
![captu2ra](https://user-images.githubusercontent.com/5629919/37853565-31205a8e-2edf-11e8-87b4-7abea5205eb7.JPG)

![captura](https://user-images.githubusercontent.com/5629919/37853544-1dfd7bee-2edf-11e8-880f-06c9f6040fc1.JPG)

**Comment by erperejildo** [comment]
Also tried here:
![captura](https://user-images.githubusercontent.com/5629919/37858615-0d7a5902-2eff-11e8-94aa-d3d8d83ea14e.JPG)
![captura2](https://user-images.githubusercontent.com/5629919/37858616-11dc8592-2eff-11e8-85e8-62f68a3cfab8.JPG)
https://javebratt.com/ionic-google-login/
